pom.xml

```diff
@@ -54,7 +54,17 @@
         </configuration>
       </plugin>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
There is a property defined in parent pom called `integration-test-phase` which for now specified only for `docsBuild` profile, probably we need to change that instead.